packages/cli-core/GitUtilities.js

```diff
@@ -74,26 +80,32 @@ exports.assertMatchesRemote = async () => {
 
   if (result !== '0')
     throw new Error('The remote differs from the local working tree');
-};
+}
 
-exports.addRemote = (dest, name, org) =>
-  execa('git', ['remote', 'add', 'origin', exports.remoteUrl(name, org)], {
+export function addRemote(dest, name, org) {
+  return execa('git', ['remote', 'add', 'origin', remoteUrl(name, org)], {
     cwd: dest,
     stdio: 'inherit',
   }).catch((err) => {
     if ((err.stderr || '').match(/remote origin already exists/)) return;
     throw err;
   });
+}
 
-exports.addTag = async (tag) => {
+export async function addTag(tag) {
   if (await hasTag(tag)) return;
   await execa('git', ['tag', tag, '-m', tag]);
-};
+}
 
-exports.removeTag = (tag) => execa('git', ['tag', '-d', tag]);
-exports.getCurrentBranch = (opts) =>
-  execa('git', ['rev-parse', '--abbrev-ref', 'HEAD'], opts).then(
+export function removeTag(tag) {
+  return execa('git', ['tag', '-d', tag]);
+}
+export function getCurrentBranch(opts) {
+  return execa('git', ['rev-parse', '--abbrev-ref', 'HEAD'], opts).then(
```

Comment:
any chance while we're touching this we can move from execa? might be better in a separate PR

Reply:
I don't want to move more things than necessary for this. i'm already pretty unsure i didn't break anything